oak-core/src/main/java/org/apache/jackrabbit/oak/plugins/migration/NodeStateCopier.java

```diff
@@ -308,6 +316,8 @@ private static NodeBuilder getChildNodeBuilder(@NotNull final NodeBuilder root,
 
         private Set<String> mergePaths = emptySet();
 
+        private boolean referenceableFrozenNodes = true;
```

Comment:
Was thinking if this should rather be `false` here - as that's the new default. But haven't checked all implications..

Reply:
That's exactly the reason why I set the default to true. I was worried there may be code out there using this class without us knowing. Setting it to false by default would then change the behaviour.